include/server/SC_WorldOptions.h

```diff
@@ -29,74 +29,62 @@
 
 typedef int (*PrintFunc)(const char *format, va_list ap);
 
-struct WorldOptions
+const struct WorldOptions
```

Comment:
this is a really long struct definition. IMO clearer if you define the struct type here and then define the const instance below it.